config/gradle/jacoco.gradle

```diff
@@ -106,10 +115,14 @@ ext.coverageExclusions = [
     '**/SecurityCodeInput.*',
     '**/SocialSecurityNumberInput.*',
 
-    // Test classes
-    '**/*Test.*',
-    '**/Test*.*',
+    // Project specific files that can't be unit tested
```

Comment:
What about excluding `View`s as well for now until we have proper testing?

Reply:
We already did, but I just pushed something to make it work better